File: 27-remove-element/remove-element.cpp
class Solution {
public:
int removeElement(vector<int>& nums, int val) {
    int i=0;
    while (i<nums.size()){
       if (val==nums[i]){
           nums.erase(nums.begin()+i);
       
   
          i--;
   
       };
       i++;
   
    };
    return nums.size();  
}
};

File: 27-remove-element/remove-element.py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        nr=[]
       
        for i in range(len(nums)):
            if val != nums[i]:
                nr.append(nums[i])
               
      
        nums[:] = nr + ['_'] * (len(nums) - len(nr))

        return len(nr)
            

